lib/features/composer/presentation/composer_controller.dart

```diff
@@ -438,8 +433,8 @@ class ComposerController extends BaseController {
     if (identitySelected.value?.replyTo?.isNotEmpty == true) {
       listReplyToEmailAddress = identitySelected.value!.replyTo!;
     }
-    final generatePartId = PartId(_uuid.v1());
-    final generateBlobId = Id(_uuid.v1());
+    final generatePartId = PartId(const Uuid().v1());
```

Comment:
Why you need to change that? IMO, you will have trouble with it in test, can not mock, test it

Reply:
We don't need to worry about it. Since the controller it manages quite a lot, writing unit tests is not necessary here. On the other hand `Uuid ` is not too complicated so we don't need to create `singleton` for it

Reply:
Ok, I made a mistake here, I'll fix it